framework/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_subdirectory(actions)
 add_subdirectory(shortcuts)
 add_subdirectory(workspace)
 add_subdirectory(network)
+add_subdirectory(system)
```

Comment:
What else can you put into a systems module?
I am confused that the modules of the network, system are very small
Maybe they should be merged?

Reply:
Or maybe fsoperation can be moved to the global?

Reply:
decided to merge these modules, remove the network module in another pull request.